examples/fluent-bit/multi-config-support/README.md

```diff
@@ -0,0 +1,83 @@
+## FireLens Example: Multiple Config support - using the Fluent Bit image with init tag
+
+This example shows you how to set multiple config files for Fluent Bit on ECS. 
+
+This example simulates a situation: A ECS Task include two containers, the app container using httpd image, and the log router container using Fluent Bit image, which will forward the logs from the App container to CloudWatch. 
+
+Based on above situation, we want to use multiple config feature to parse dummy data logs first, and forward the parsed dummy data logs to S3.
+
+
+
+### Step 1: Create config files locally
+
+**dummy-input.conf**
+
+```
+[INPUT]
```

Comment:
configuration value row is not align as a row, please reformat this part.

Reply:
You can also use the button on the right top corner beside `< >` which is `Display the rich diff` to look how the markdown file looks like.